app/livechat/client/views/app/livechatCurrentChats.js

```diff
@@ -249,13 +256,18 @@ Template.livechatCurrentChats.events({
 Template.livechatCurrentChats.onCreated(async function() {
 	this.ready = new ReactiveVar(false);
 	this.limit = new ReactiveVar(20);
-	this.filter = new ReactiveVar({});
 	this.selectedAgents = new ReactiveVar([]);
 	this.customFilters = new ReactiveVar([]);
 	this.customFields = new ReactiveVar([]);
 	this.tagFilters = new ReactiveVar([]);
 	this.departments = new ReactiveVar([]);
 
+	if (localStorage.getItem('status')) {
```

Comment:
Prevent code duplicationg by moving the getItem from localstorage to a variable.
Use a better name than just `status`, it's too generic and may be used by another place in the application. Use some thing like `livechatCurrentChats.filter.status`